src/components/Target/TargetForm.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import FormField from '../common/FormField';
+import '../../styles/Common.scss';
+import Select from 'react-select';
+
+class TargetForm extends Component {
+  constructor() {
+    super();
+    this.state = { value: 'two' };
+    this.toggle = this.toggle.bind(this);
+  }
+  toggle({ value }) {
+     this.setState({ value });
+  }
+
+  render() {
+    const { handleSubmit } = this.props;
+    let options = [
```

Comment:
This shouldn't be in the `render` function, use a function to return them or a `const`